gnoi/gnoi.go

```diff
@@ -33,14 +34,15 @@ var (
 	rsaBitSize = 2048
 )
 
-// Server does blah.
+// Server represents a target
```

Comment:
Thanks for cleaning this up. Go comments should be full sentences ending with a period.